include/crm/msg_xml_compat.h

```diff
@@ -406,128 +406,8 @@ extern "C" {
-
-//! \deprecated Do not use
-#define F_CRM_ELECTION_ID "election-id"
-
-//! \deprecated Do not use
-#define F_CRM_ELECTION_AGE_S "election-age-sec"
-
-//! \deprecated Do not use
-#define F_CRM_ELECTION_AGE_US "election-age-nano-sec"
-
-//! \deprecated Do not use
-#define F_CRM_ELECTION_OWNER "election-owner"
-
-//! \deprecated Do not use
-#define F_CRM_TGRAPH "crm-tgraph-file"
-
-//! \deprecated Do not use
-#define F_CRM_TGRAPH_INPUT "crm-tgraph-in"
-
-//! \deprecated Do not use
-#define F_CRM_THROTTLE_MODE "crm-limit-mode"
-
-//! \deprecated Do not use
-#define F_CRM_THROTTLE_MAX "crm-limit-max"
-
-//! \deprecated Use \c PCMK_XA_RESULT instead
-#define XML_PING_ATTR_STATUS PCMK_XA_RESULT
-
-//! \deprecated Do not use
-#define XML_PING_ATTR_SYSFROM "crm_subsystem"
-
-//! \deprecated Do not use
-#define XML_PING_ATTR_CRMDSTATE "crmd_state"
-
-//! \deprecated Do not use
-#define XML_PING_ATTR_PACEMAKERDSTATE "pacemakerd_state"
-
-//! \deprecated Do not use
-#define XML_FAILCIB_ATTR_OBJTYPE "object_type"
-
-//! \deprecated Use \c PCMK_XA_OPERATION instead
-#define XML_FAILCIB_ATTR_OP PCMK_XA_OPERATION
-
-//! \deprecated Use \c PCMK_XA_OPERATION instead
-#define XML_LRM_ATTR_TASK PCMK_XA_OPERATION
-
-//! \deprecated Use \c PCMK_XA_OPERATION instead
-#define XML_EXPR_ATTR_OPERATION PCMK_XA_OPERATION
-
-//! \deprecated Use \c PCMK_XA_OPERATION instead
-#define XML_DIFF_OP PCMK_XA_OPERATION
-
-//! \deprecated Use \c PCMK_XA_REASON instead
-#define XML_FAILCIB_ATTR_REASON PCMK_XA_REASON
-
-//! \deprecated Use \c PCMK_META_TIMEOUT instead
-#define XML_ATTR_TIMEOUT PCMK_META_TIMEOUT
-
-//! \deprecated Use \c PCMK_META_TIMEOUT instead
-#define XML_ALERT_ATTR_TIMEOUT PCMK_META_TIMEOUT
-
-//! \deprecated Use \c PCMK_XA_PATH instead
-#define XML_ALERT_ATTR_PATH PCMK_XA_PATH
-
-//! \deprecated Use \c PCMK_XA_PATH instead
-#define XML_DIFF_PATH PCMK_XA_PATH
-
-//! \deprecated Use \c PCMK_META_TIMESTAMP_FORMAT instead
-#define XML_ALERT_ATTR_TSTAMP_FORMAT PCMK_META_TIMESTAMP_FORMAT
-
-//! \deprecated Use \c PCMK_META_INTERVAL instead
-#define XML_LRM_ATTR_INTERVAL PCMK_META_INTERVAL
-
-//! \deprecated Use \c PCMK_META_INTERVAL instead
-#define XML_LRM_ATTR_INTERVAL_MS PCMK_META_INTERVAL
-
-//! \deprecated Do not use
-#define XML_CIB_ATTR_REPLACE "replace"
-
-//! \deprecated Do not use
-#define XML_COLOC_ATTR_SOURCE_INSTANCE "rsc-instance"
-
-//! \deprecated Do not use
-#define XML_COLOC_ATTR_TARGET_INSTANCE "with-rsc-instance"
-
-//! \deprecated Use \c PCMK_META_ON_FAIL instead
-#define XML_OP_ATTR_ON_FAIL PCMK_META_ON_FAIL
-
-//! \deprecated Use \c PCMK_META_START_DELAY instead
-#define XML_OP_ATTR_START_DELAY PCMK_META_START_DELAY
-
-//! \deprecated Use \c PCMK_META_INTERVAL_ORIGIN instead
-#define XML_OP_ATTR_ORIGIN PCMK_META_INTERVAL_ORIGIN
-
-//! \deprecated Use \c PCMK_META_RECORD_PENDING instead
-#define XML_OP_ATTR_PENDING PCMK_META_RECORD_PENDING
+//! \deprecated Use \c PCMK_XE_PARAMETERS instead
+#define XML_TAG_PARAMS PCMK_XE_PARAMETERS
```

Comment:
Looks like this accidentally deleted a whole bunch of stuff (probably in rebase)

Reply:
Oops